docs/api-docs/storefront/graphql/graphql-cart-metafields.mdx

```diff
@@ -0,0 +1,328 @@
+{
+  "data": {
+    "site": {
+      "products": {
+        "edges": [
+          {
+            "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
+            "node": {
+              "metafields": {
+                "edges": []
+              },
+              "variants": {
+                "edges": [
+                  {
+                    "node": {
+                      "sku": "SLCTBS",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
+            "node": {
+              "metafields": {
+                "edges": []
+              },
+              "variants": {
+                "edges": [
+                  {
+                    "node": {
+                      "sku": "s-1138-SM-RE-RE",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  },
+                  {
+                    "node": {
+                      "sku": "s-1138-SM-GR-RE",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  },
+                  {
+                    "node": {
+                      "sku": "s-1138-SM-BL-RE",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  },
+                  {
+                    "node": {
+                      "sku": "s-1138-ME-RE-RE",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  },
+                  {
+                    "node": {
+                      "sku": "s-1138-ME-GR-RE",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
+            "node": {
+              "metafields": {
+                "edges": []
+              },
+              "variants": {
+                "edges": [
+                  {
+                    "node": {
+                      "sku": "",
+                      "metafields": {
+                        "edges": []
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+</Tab>
+</Tabs>
+
+
+### Create a cart metafield
+
+<Callout type="info">
+The current platform limit is 250 metafields for a single cart. 
+</Callout>
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    
+  ```graphql filename="Example mutation: Create a cart metafield" showLineNumbers copy 
```

Comment:
the syntax highlighting makes it look like something is off in this block.

In addition, can you separate the variables being sent into a json object in a third tab? That's how we advise people to construct queries.

Reply:
@slsriehl I think I fixed the spacing. We don't need another tab for variables because the variables are already in the request.